tasks/install-matlab/v0/test/downloadAndExecuteLinux.ts

```diff
@@ -13,11 +13,13 @@ process.env.SYSTEM_SERVERTYPE = "hosted";
 
 tr.registerMock("azure-pipelines-tool-lib/tool", {
     downloadTool(url: string) {
-        if (url !== "https://ssd.mathworks.com/supportfiles/ci/matlab-deps/v0/install.sh"
-            && url !== "https://ssd.mathworks.com/supportfiles/ci/ephemeral-matlab/v0/install.sh") {
+        if (url === "https://ssd.mathworks.com/supportfiles/ci/matlab-deps/v0/install.sh") {
+            return "install.sh";
```

Comment:
Given the change in `tasks/install-matlab/v0/main.ts`, are there modes where just `install.sh` would be used and not `ci-install.sh` 🤔

Reply:
This is the matlab-deps url, which has kept the install.sh name.

Reply:
Ahhhh oops that makes sense; I forgot about matlab-deps 😂